packages/data-mate/src/interfaces.ts

```diff
@@ -0,0 +1,14 @@
+import { FieldTypeConfig } from '@terascope/data-types';
+
+export type ArgSchema = Config & { describe?: string };
```

Comment:
Should we be consistent with the descriptions?

```suggestion
export type ArgSchema = Config & { description?: string };
```